docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Managing resources using scripts
-description: Octopus resources can be created using service messages allowing resources that you currently can script in Azure to be modeled in Octopus (dynamic infrastructure).
+title: Dynamic infrastructure
+description: Octopus resources created in cloud providers can also be modeled in Octopus using service messages and scripts, which form part of dynamic infrastructure.
 position: 120
 ---
 
-Some resources can be created within Octopus from the same scripts that you use to create them on Azure. By adding some additional commands Web Apps you create on Azure can also be created within Octopus as deployment targets.
+You can use the [Octopus REST API](/docs/octopus-rest-api/index.md) or the Octopus commands below to create Octopus accounts, targets, and workers dynamically. You can make these requests in the same scripts that create your cloud infrastructure or in following steps.
 
 :::success
-Only Azure Service Principal Accounts, Azure Web Apps, Azure Service Fabric, Azure Cloud Services and Kubernetes targets are supported.
+Only a subset of account types and deployment targets support being created dynamically. The ones which are supported are listed below.
```

Comment:
This is only true if you're using the commandlets, with the API you can create anything right?

Reply:
Correct, I'll add some clarification to that statement now I've added the API section

Reply:
Updated the success to a warning, and moved closer to the section it refers to (and added reference to the commands in) 1349e6c31247d60a26669b0e4389a541589db5aa